### ISSUE TITLE
Backup passwords. 

### ISSUE THREAD

**Issue by ITronlegendsoffical** [OffRange/KeyGo#6]
**pretty obvious, make a custom file to store passwords.**

**Description:**
Add a file password export options via a custom file.

**Problem/Use Case:**
Used to export passwords to a 2nd device.

**Benefits:**
Obvious benefits. Check use cases.

**Alternatives: **
Maybe a cloud option?

**Additional Context **
It might be a good idea to ask for the master password when making/ importing the file.

**Comment by OffRange** [comment]
Indeed, it's an excellent enhancement! I'm already considering implementing it, potentially in release 1.2 later this year. Moreover, with that release, I also consider launching a desktop version if everything goes as planned.

**Comment by mylespm** [comment]
How about support for KeePass (.kdbx)?
